adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBRewardedlRenderer.m

```diff
@@ -0,0 +1,117 @@
+//
+//  GADPangleRTBRewardedlRenderer.m
+//  PangleAdapter
+//
+//  Created by bytedance on 2022/1/11.
+//
+
+#import "GADPangleRTBRewardedlRenderer.h"
+#import "GADMediationAdapterPangleConstants.h"
+#import <BUAdSDK/BUAdSDK.h>
+#import "GADMAdapterPangleUtils.h"
+
+@interface GADPangleRTBRewardedlRenderer()<BURewardedVideoAdDelegate>
+
+@property (nonatomic, strong) GADMediationRewardedAdConfiguration *adConfig;
+@property (nonatomic, copy) GADMediationRewardedLoadCompletionHandler loadCompletionHandler;
+
+@property (nonatomic, strong) BURewardedVideoAd *rewardedVideoAd;
+
+@property (nonatomic, weak) id<GADMediationRewardedAdEventDelegate> delegate;
```

Comment:
Prefer all these to be instance variables rather than properties. The same applies to other renderer classes.